examples/quickstart_etl/README.md

```diff
@@ -1,8 +1,18 @@
 # Dagster starter kit
 
-This example is a starter kit for building a daily ETL pipeline. At a high level, this project shows how to ingest data from external sources, explore and transform the data, and materialize outputs that help visualize the data.
+This example project is a starting point for building a data engineering pipeline.
 
-*New to Dagster? Learn what Dagster is [in Concepts](https://docs.dagster.io/concepts) or [in the hands-on Tutorials](https://docs.dagster.io/tutorial).*
+This project follows an important pattern in data engineering: ETL (Extract-Transform-Load). The ETL pipeline in this project:
+
+- Extracts data from an external source
+- Transforms the data to be easier to explore
+- Loads the transformed data into a report to visualize it.
+
+To productionize this project, the pipeline is scheduled to run daily and is ready to be deployed to Dagster Cloud of your self-hosted Dagster instance.
```

Comment:
Not really a 🔧 , but not really a 🌱 

Maybe ✏️ ? Either way, consider linking to the docs here. There's also a small typo (`... to Dagster Cloud OR your self-hosted Dagster instance.`)

Ex:

```suggestion
To productionize this project, the pipeline is scheduled to run daily and is ready to be deployed to [Dagster Cloud](https://docs.dagster.io/dagster-cloud) or your [self-hosted Dagster instance](https://docs.dagster.io/deployment#deploying-to-your-infrastructure).
```